trillian/log_server/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM scratch
+ADD main /
+VOLUME ["/config"]
+ENV DB_USER root
+ENV DB_PASS=
+ENV DB_HOST db
+ENV DB_PORT 3306
```

Comment:
I don't think the ct server container would be able to access the port, without it becoming explicitly exposed in the Dockerfile.